tests/acceptance/features/webUISharingInternalUsersToRoot/shareWithUsers.feature

```diff
@@ -10,7 +10,7 @@ Feature: Sharing files and folders with internal users
       | Alice    |
       | Brian    |
 
-  @smokeTest @disablePreviews @skipOnXGAPortraitResolution
```

Comment:
Why are these reintroduced to the portraitResolution tests? The linked/fixed issue is only about `tests/acceptance/features/webUIDeleteFilesFolders/deleteFilesFolders.feature`, not this test

Reply:
Previously, they failed exactly due to the same issue. So, I thought it would be good to un-skip them back.

Reply:
Some detail here: https://github.com/owncloud/web/issues/6594#issuecomment-1067800496